File: c/arrays/arrays_1d/array_creation_1.c
#include<stdio.h>
int main(){
    int n;
    printf("Enter the size of arrays : ");
    scanf("%d",&n);

    int arr[n];
    
    printf("Enter the elements of arrays :");
    for(int i = 0; i < n; i++){
        scanf("%d",&arr[i]);
    }

    for(int j = 0; j < n; j++){
        printf("%d",arr[j]);
    }
}

File: java/arrays/arrays_1d/ArrayCreation_1.java
import java.util.Scanner;
class ArrayCreation_1{
    public static void main(String args[]){
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the size of array :");
        n = sc.nextInt();
        int []arr = new int[n];

        System.out.println("Enter the elements of array :");
        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
        }

        for(int j = 0; j < n; j++){
            System.out.print(arr[j]+" ");
        }
    }
}